frontends/infinite-corridor/src/pages/FieldPageSkeleton.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react"
+import { BannerPage } from "ol-util"
+import Container from "@mui/material/Container"
+import { useFieldDetails } from "../api/fields"
+
+interface FieldSkeletonProps {
+  children: React.ReactNode
+  name: string
+}
+
+/**
+ * Common structure for field-oriented pages.
+ *
+ * Renders the field title and avatar in a banner.
+ */
+const FieldSkeletonProps: React.FC<FieldSkeletonProps> = ({
+  children,
+  name
+}) => {
+  const field = useFieldDetails(name)
+
+  return (
+    <BannerPage
+      compactOnMobile
```

Comment:
Add this here (or something like it) so the banner appears:

```
src={field.data?.banner ?? ""}
```

<img width="874" alt="Screen Shot 2022-07-26 at 3 34 43 PM" src="https://user-images.githubusercontent.com/187676/181096677-9b18145a-4f58-4bec-a503-9cd620b8e38d.png">